src/util/http/websocket/UpdateFetcher.cpp

```diff
@@ -9,16 +9,23 @@
 namespace ad_utility::websocket {
 
 net::awaitable<UpdateFetcher::PayloadType> UpdateFetcher::waitForEvent() {
-  if (!distributor_) {
-    distributor_ =
-        co_await queryHub_.createOrAcquireDistributorForReceiving(queryId_);
-  }
+  try {
+    if (!distributor_) {
+      distributor_ =
+          co_await queryHub_.createOrAcquireDistributorForReceiving(queryId_);
+    }
 
-  auto data =
-      co_await sameExecutor(distributor_->waitForNextDataPiece(nextIndex_));
-  if (data) {
-    nextIndex_++;
+    auto data =
+        co_await sameExecutor(distributor_->waitForNextDataPiece(nextIndex_));
+    if (data) {
+      nextIndex_++;
+    }
+    co_return data;
+  } catch (boost::system::system_error& error) {
+    if (error.code() == net::error::operation_aborted) {
```

Comment:
Leave a short comment, that this is actually a pretty normal case that we don't want to have in the log.